components/compliance-service/api/status/server/status.go

```diff
@@ -18,13 +18,14 @@ const (
 	MigrationLabelESa2v2 = "ElasticSearch_A2_v2"
 	MigrationLabelESa2v3 = "ElasticSearch_A2_v3"
 	MigrationLabelESa2v4 = "ElasticSearch_A2_v4"
+	MigrationLabelESa2v5 = "ElasticSearch_A2_v5"
 	MigrationLabelPG     = "PostgreSQL"
 	MigrationLabelPRO    = "Profiles"
 	//MigrationLabelFEEDS = "ElasticSearch_Feeds_1"
 )
 
-const maxMigrations = 7        // Total migrations should match the number of constants above
-const totalMigrationSteps = 28 // Max number of migration LogEntry items we can have across all migrations
+const maxMigrations = 8        // Total migrations should match the number of constants above
+const totalMigrationSteps = 36 // Max number of migration LogEntry items we can have across all migrations
```

Comment:
🎉 we've seen this number get out of date quite a bit - is there something we should be doing to increase it automagically or is manual the best way to go?
<img width="641" alt="Screen Shot 2019-12-03 at 12 14 36" src="https://user-images.githubusercontent.com/10341541/70081930-a2f69b80-15c6-11ea-90ca-7cd2278496a4.png">

Reply:
I agree we should find an automatic method to determine the maximum number of migration steps. It will require a bit of a refactor in another PR.